bin/docker_start.sh

```diff
@@ -37,6 +37,18 @@ if [ -d $fixtures_dir ]; then
     done
 fi
 
+# Create superuser
+# specify password by setting DJANGO_SUPERUSER_PASSWORD in the env
+# specify username by setting OPENZAAK_SUPERUSER_USERNAME in the env
+# specify email by setting OPENZAAK_SUPERUSER_EMAIL in the env
+if [ -z "${OPENZAAK_SUPERUSER_USERNAME}" ]; then
+    python src/manage.py createinitialsuperuser \
+        --no-input \
+        --username "${OPENZAAK_SUPERUSER_USERNAME}" \
+        --email "${OPENZAAK_SUPERUSER_EMAIL:-admin\@admin.org}"
+    unset OPENZAAK_SUPERUSER_USERNAME OPENZAAK_SUPERUSER_EMAIL DJANGO_SUPERUSER_PASSWORD
```

Comment:
unsetting these here, since the actual application has no need for them and we don't want any credentials in memory